.travis.yml

```diff
@@ -12,10 +12,16 @@ env:
 before_install:
 - gem install bundler
 before_script:
+- cp ./v2_key.dev ./v2_key
 - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64
   > ./cc-test-reporter
 - chmod +x ./cc-test-reporter
 - "./cc-test-reporter before-build"
-- bundle exec rake db:create db:migrate
+- bundle install
```

Comment:
Isn't the `bundle install` already done by the `install` step?

Reply:
yep, I just removed it. 👍